src/app/layout/components/header/header.component.scss

```diff
@@ -1,5 +1,16 @@
 
-#toolbarContent
+.flex {
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  align-items: center;
+
+  > .col {
+    flex: 0 0 33.33%
+  }
+}
+/* #toolbarContent
```

Comment:
//command remove comments -m "less code = less headache, even if it is a comment"